db/migrate/20230324091733_create_conditions_table.rb

```diff
@@ -0,0 +1,11 @@
+class CreateConditionsTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :conditions do |t|
+      t.references :check_page
```

Comment:
is it useful add `:comment` to each of the references describing what role it plays (e.g. from your PR intro)

for example: 
```t.references :check_page, comment: "The page this condition is attached to"```

We don't do that in other places but it might make things a bit clearer for those infra devs mucking around in the db

Reply:
This is a nice idea I think!